sha2/src/sha256.rs

```diff
@@ -157,6 +157,9 @@ cfg_if::cfg_if! {
         mod soft;
         mod aarch64;
         use aarch64::compress;
+    } else if #[cfg(all(target_os = "zkvm", target_vendor = "succinct", target_arch = "riscv32"))] {
```

Comment:
Our crates aim for algorithm implementation _in Rust_, so I don't think we will use OS-specific crypto APIs like you did in this PR.

Reply:
Mistakenly put these PR's up against the main repo, that makes sense!